api/src/main/java/jakarta/enterprise/concurrent/ManagedExecutorDefinition.java

```diff
@@ -213,6 +220,12 @@
      * In situations such as these, the executor does not control
      * the type of thread that is used to run the task.</p>
      *
+     * <p>The {@code true} value is permitted only when running on
```

Comment:
I suggest to change this to something like behavior is unspecified when virtual threads are not supported by the JVM.

Reply:
I am with @OndroMih on this one to use the category of `the behavior is unspecified` or `non-portable behavior might occur`. I think it is wrong to throw an error because `true` is Not an invalid value. Even in Java 21, JVM might not give you a VT. Nothing is wrong for asking. The worst case scenario is not to have one even though you have asked one when running at JDK 21-.